_episodes/04-performance.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: "Teaching as a Performance Art"
```

Comment:
Yup - I think that 90% or more of our trainees are grad students/post-docs, and this is their first exposure to *any* training in teaching, so the idea of "teaching as performance" will be new to them.